README.md

```diff
@@ -36,7 +36,7 @@ git submodule update --init --recursive --depth 1
 
 ## Running the website using a container
 
-To build the site in a container, run the following to build the container image and run it:
+In order to build the site in a container, run the following to build the container image:
```

Comment:
See the [Kubernetes style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language) about using simple and direct language
```suggestion
In order to build the site in a container, run the following to build the container image:
```
With the example in the style guide, I prefer the original